newdle/client/src/components/Navigator.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
`npm i --save prop-types` inside newdle/client/ - it just works right now because prop-types is installed as a dependency of something; but if we use a library it should be in our package.json